src/XRFeitoriaBpy/core/factory.py

```diff
@@ -133,29 +145,35 @@ def set_sequence_properties(
             frame_start (int): Start frame of the sequence.
             frame_end (int): End frame of the sequence.
             frame_current (int): Current frame of the sequence.
+            resoltion_x (int): Resolution_x of the sequence.
+            resoltion_y (int): Resolution_y of the sequence.
         """
         collection.sequence_properties.level = level
         collection.sequence_properties.fps = fps
         collection.sequence_properties.frame_start = frame_start
         collection.sequence_properties.frame_end = frame_end
         collection.sequence_properties.frame_current = frame_current
+        collection.sequence_properties.resolution_x = resolution_x
+        collection.sequence_properties.resolution_y = resolution_y
 
-    def get_sequence_properties(collection: 'bpy.types.Collection') -> 'Tuple[bpy.types.Scene, int, int, int, int]':
+    def get_sequence_properties(collection: 'bpy.types.Collection') -> SequenceProperties:
         """Get the sequence properties.
 
         Args:
             collection (bpy.types.Collection): Collection of the sequence.
 
         Returns:
-            Tuple[bpy.types.Scene, int, int, int, int]:
-                The level(scene), FPS of the sequence, Start frame of the sequence, End frame of the sequence, Current frame of the sequence.
+            SequenceProperties: Sequence properties.
         """
         level = collection.sequence_properties.level
         fps = collection.sequence_properties.fps
         frame_start = collection.sequence_properties.frame_start
         frame_end = collection.sequence_properties.frame_end
         frame_current = collection.sequence_properties.frame_current
-        return level, fps, frame_start, frame_end, frame_current
+        resolution_x = collection.sequence_properties.resolution_x
+        resolution_y = collection.sequence_properties.resolution_y
+
+        return level, fps, frame_start, frame_end, frame_current, resolution_x, resolution_y
```

Comment:
A class inheriting NamedTuple is not only a typing, but can also be used to initialize an instance.

```suggestion
        return SequenceProperties(level, fps, frame_start, frame_end, frame_current, resolution_x, resolution_y)
```

Reply:
Accepted in [`6ec2877` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/6ec287755d8d27ec5a11e8578c65e2581ffc02c9)